pkg/compose/watch.go

```diff
@@ -37,8 +37,8 @@ import (
 	"github.com/docker/docker/api/types/container"
 	"github.com/docker/docker/api/types/filters"
 	"github.com/docker/docker/api/types/image"
+	"github.com/go-viper/mapstructure/v2"
```

Comment:
You should keep "github.com/mitchellh/mapstructure"

Reply:
It's strange, I haven't changed it manually, it looks like IDE did it. Fixed